src/components/Logo.tsx

```diff
@@ -1,13 +1,20 @@
-export default function Example() {
+import { useState } from "react"
+
+export const Logo = () => {
+    const [logo, setLogo] = useState(true);
+
+    const clicked = () => {
+        setLogo(!logo);
+    }
+
     return (
         <>
             <div>
                 <div className="flex items-center justify-center relative">
                     <div className="p-1 z-50">
```

Comment:
I would do `z-40` so that it doesn't go over your navbar

Reply:
No matter what I set my z to it goes over the navbar